app/views/sessions/new.html.haml

```diff
@@ -0,0 +1,11 @@
+%h1 Log In
+= form_for :session, :url => "/login" do |f|
+  = f.label :email
+  %br
+  = f.text_field :email
```

Comment:
@njgheorghita Are we using email or phone number for authentication?  As the User model is right now, the email field isn't required, so we could have folks creating users with no email address - but then we wouldn't be able to find them in the database... It seems like that's something (email vs. phone_number) that we might need to figure out as a group.